svm/src/loader.rs

```diff
@@ -29,6 +31,40 @@ pub trait Loader {
             .and_then(|account| owners.iter().position(|entry| account.owner() == entry))
     }
 
+    /// Collect rent from an account if rent is still enabled and regardless of
+    /// whether rent is enabled, set the rent epoch to u64::MAX if the account is
+    /// rent exempt.
+    ///
+    /// This function has a default implementation, but projects can override
+    /// it if they want to provide a custom implementation.
+    fn collect_rent_from_account(
+        &self,
```

Comment:
`self` is not really used here, right? Wondering if this should really be in this trait.